### PR TITLE
Add config for layer c.bafu.bundesinventare-bln

### PR DESCRIPTION
**Pull request by alcapat**
Use table update for the moment because table structure has changed

### CODE REVIEW

File: conf/bafu.conf.part [hunk 1]
```diff
@@ -750,6 +750,23 @@ source src_ch_bafu_karst-ausdehnung_grundwasservorkommen : def_searchable_featur
...
+        SELECT bgdi_id::bigint as id \
+            , bln_name as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(bln_obj::text, ' ')||' '||coalesce(bln_name, ' ')||' '||coalesce(subareaname, ' ')) as  detail \
+            , 'ch.bafu.bundesinventare-bln' as layer \
+            , quadindex(the_geom) as geom_quadindex \
+            , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+            , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+            , box2d(the_geom) as geom_st_box2d \
+            , bgdi_id::text as feature_id \
```
Comment (ltclm) [inline conf/bafu.conf.part]: im not sure but i think feature_id should be the same column used by the model as primary key
https://github.com/geoadmin/mf-chsdi3/blob/68e20ec0004cbc0aed322c7c14da72c02ea9d17f/chsdi/models/vector/bafu.py#L1217
@loicgasser do you agree?

Comment (loicgasser) [inline conf/bafu.conf.part]: yes it just changed.

Comment (loicgasser) [inline conf/bafu.conf.part]: @alcapat please change the target table as we created a new one.

Comment (ltclm) [inline conf/bafu.conf.part]: @alcapa please remove the replaced table afterwards.

**Pull request merged**